### PR TITLE
Tidy Notch filter logging

### PR DESCRIPTION
**Pull request by peterbarker**
This was a bit weird, with the vehicle calling the library and the library calling the vehicle.  This makes it less weird.

Tested in SITL by running the GyroFFT test and checking the logs still contained the `FTN` message.

Better still would be to pass the log-bitmask-bit (e.g. `MASK_LOG_CTUN`) into the AP_InertialSensor library and have it do the relevant logging.  It's a pattern we use elsewhere.

### CODE REVIEW

**Review by andyp1per — APPROVED** [review]
It was weirder before

**Review by hendjoshsr71 — APPROVED** [review]
LGTM.

File: libraries/AP_InertialSensor/AP_InertialSensor_Logging.cpp [hunk 1]
```diff
@@ -128,3 +128,29 @@ bool AP_InertialSensor::BatchSampler::Write_ISBD() const
...
+// @Field: NF7: dynamic harmonic notch centre frequency for motor 7
+// @Field: NF8: dynamic harmonic notch centre frequency for motor 8
+// @Field: NF9: dynamic harmonic notch centre frequency for motor 9
+// @Field: NF10: dynamic harmonic notch centre frequency for motor 10
+// @Field: NF11: dynamic harmonic notch centre frequency for motor 11
+// @Field: NF12: dynamic harmonic notch centre frequency for motor 12
+void AP_InertialSensor::write_notch_log_messages() const
+{
+    const float* notches = get_gyro_dynamic_notch_center_frequencies_hz();
+    AP::logger().Write(
```
Comment (tridge) [inline libraries/AP_InertialSensor/AP_InertialSensor_Logging.cpp:151]: should be WriteStreaming

**Review by tridge — CHANGES_REQUESTED** [review]

File: libraries/AP_InertialSensor/AP_InertialSensor_Logging.cpp (hunk 1 above)
Comment (peterbarker) [reply libraries/AP_InertialSensor/AP_InertialSensor_Logging.cpp]: Done

**Pull request merged**